settings/chicago/chicago_baseCase.yml

```diff
@@ -149,9 +151,9 @@ partnership:
           prob: 1.0
           min: 37
           max: 48
-  sex:
+  pca:
     frequency:
```

Comment:
should pca frequency options match the sex ones? (with both bins or distribution as wanted?)

Reply:
Potentially, but I'm hoping to get away from use of bins in favor of distributions.